collect_app/src/main/java/org/odk/collect/android/views/CustomSpinner.java

```diff
@@ -0,0 +1,60 @@
+package org.odk.collect.android.views;
+
+import android.content.Context;
+import android.support.v7.widget.AppCompatSpinner;
+import android.util.AttributeSet;
+
+/**
+ * Created by vaibhav on 1/3/18.
+ */
```

Comment:
Please replace this comment with a description of the class. The very top should have a copyright notice with your name (you are the copyright holder) -- https://github.com/opendatakit/collect/blob/master/LICENSE.md#appendix-how-to-apply-the-apache-license-to-your-work